src/Connection.php

```diff
@@ -42,7 +42,7 @@ class Connection
 
 
 	/**
-	 * @param  array $config see drivers for supported options
+	 * @param  array        $config       see drivers for supported options
```

Comment:
yeah what I meant was it was typo that I changed it as I was experimenting with implemenations